README.md

```diff
@@ -1,29 +1,54 @@
+
 # About the project
 
-> You can download an [offline version](https://www.gitbook.com/book/openfisca/documentation) of the documentation (direct link to [PDF](https://www.gitbook.com/download/pdf/book/openfisca/documentation), [ePub](https://www.gitbook.com/download/epub/book/openfisca/documentation) and [Mobi](https://www.gitbook.com/download/mobi/book/openfisca/documentation)).
+[OpenFisca](https://www.openfisca.fr/) is a versatile microsimulation software.
+OpenFisca allows users to :
+- Calculate many variables of the tax and benefit system of a country (social benefits and taxes paid by households) given input variables.
+- Simulate the budgetary consequences of a reform and its distributional impact when plugged on a survey.
+Its engine is independent of the country, it is therefore possible to simulate any country.
+For now the main supported country is France.
 
-[![OpenFisca logo](https://www.openfisca.fr/hotlinks/logo-openfisca.svg)](https://www.openfisca.fr/)
+[OpenFisca](https://www.openfisca.fr/) is free and is published under the [GNU Affero General Public Licence](https://www.gnu.org/licenses/agpl.html) version 3 or later.
 
-[OpenFisca](https://www.openfisca.fr/) is a versatile microsimulation software.
+## Project Components
 
-It is a free software published under the [GNU Affero General Public Licence](https://www.gnu.org/licenses/agpl.html) version 3 or later.
+OpenFisca is a modular project. Depending on your goals, you will install and interact with one or several of the OpenFisca Components.
 
-## What's the purpose?
+[![OpenFisca schema](https://cdn.rawgit.com/openfisca/openfisca-doc/eb7b9ce6f41d7ab2cb7bac08738cfb67d58c0c36/architecture.svg)](https://github.com/openfisca/openfisca-doc/blob/bird_eye_view/architecture.svg)
 
-OpenFisca allows users to calculate many variables of the tax and benefit legislation of a country (social benefits and taxes paid by households) given input variables.
-The impact of reform projects can also be simulated.
+### Web API
+The Web API lets you access the legislation Parameters and Variables.
 
-When plugged on a survey, OpenFisca can also calculate the budgetary consequences of a reform
-and its distributional impact.
+>**Example**: [Mes Aides](https://mes-aides.gouv.fr) uses the OpenFisca Web API to calculate OpenFisca-France benefits.
 
-Its engine is independent of the country, it is therefore possible to simulate any country.
-For now the main supported country is France.
+- To explore the OpenFisca-France Web API services, use the [French Legistation Explorer](https://legislation.openfisca.fr/)
+- To query the Openfisca Web API in your app, see our [Web API endpoints description](./openfisca-web-api/endpoints.md)
```

Comment:
The “our” → “the” change was suggested for all added elements, not only the line on which the comment was done :wink: